src/frontend/src/components/irregularity.ts

```diff
@@ -0,0 +1,48 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon, closeIcon } from "./icons";
+
+interface irregularityProps {
+  message: string | TemplateResult;
+  additionalClasses?: string[];
+  type?: "warning" | "info" | "error";
+  closeFn?: () => void;
+}
+
+export const irregularity = ({
```

Comment:
Can you add a line or two about what this does? Also where does the term `irregularity` come from? Is "irregularity" applicable to `type = "info"` too?

Reply:
I did add a comment about it, let be know if this is ok for you or if you need a better explanation.